lib/embed.dart

```diff
@@ -573,7 +581,8 @@ class Embed {
       if (gistId.isNotEmpty) {
         gist = await loader.loadGist(gistId);
       } else if (sampleId.isNotEmpty) {
-        gist = await loader.loadGistFromAPIDocs(sampleId);
+        final useMaster = sampleChannel?.toLowerCase() == 'master';
+        gist = await loader.loadGistFromAPIDocs(sampleId, useMaster);
```

Comment:
Why not pass the string 'master' if `sampleChannel` is 'master', and 'stable' if it isn't.  Obviously we don't want to just pass through the query param, but if the argument is a string, `loadGistFromAPIDocs` will be more future proof if we do add more channels later.

Reply:
I thought about that, but figured that if we're unlikely to need more than the two choices, passing a boolean was cleaner than passing a string.

Is it more idiomatic to Dart to use a string there?

Reply:
Well, most idiomatic would probably be an enum, which would also help make sure no cases are missed in the future.

Reply:
Discussed offline with @johnpryan. We're going to need an enum for SDK channel eventually, so I've gone that route.